recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <dt-list-item
+    ref="FeedItemRef"
+    element-type="li"
+    navigation-type="tab"
+    class="d-w100p d-box-border d-ps-relative d-px16 d-py4"
+    :class="{ 'd-bgc-secondary-opaque': isActive }"
```

Comment:
so this does not override the dt-list-item--highlighted class from the DTListItem. Should we instead change the highlight color on the parent component, or provide a reactive class that adds this highlighted color instead of the default one.

ListItem code to be modified: https://github.com/dialpad/dialtone-vue/blob/69b3c5c2db67547c567845cf9b26e869d8b388e3/components/list_item/list_item.vue

CC: @braddialpad

Reply:
I'm not sure exactly what is the intended behaviour here. I feel like the list item should not highlight at all unless isActive is true, as in it doesn't automatically highlight on hover. am I correct on that?

If that's the case we might want to actually set navigation-type="none" on the list item

Reply:
oh you are right, i initially didnt get your answer properly, isActive as a prop should control everything from displaying menu actions, highlighting the feed item and not be controlled by CSS as mentioned in JIRA, we dont need to talk to design on this.